### DISCUSSION TITLE
Interpreting QC metrics within clusters

### DISCUSSION THREAD

**Discussion by wudustan** [stuart-lab/signac#500] [Q&A] [answered]
I have merged 6 scATAC datasets of different lines for comparison. I ran through the relevant vignettes in Signac.

Generating QC violin plots for the merged data I got this: 
<img width="1379" alt="Screenshot 2021-03-09 at 14 02 59" src="https://user-images.githubusercontent.com/15104192/110482136-3bd86880-80e0-11eb-8c17-60fe9a721da2.png">

Based on my results I chose the following parameters for filtering:

peak_region_fragments > 5000
peak_region_fragments < 100000
pct_reads_in_peaks > 15
blacklist_ratio < 0.05
nucleosome_signal < 4
TSS.enrichment > 2

After filtering, I generated UMAP plots of the QC metrics to see how they are distributed across the groups.

1) Looking at **peak_region_fragments**, I notice that the clusters have 'foci' of high values while the rest of the cluster can be relatively low - how I interpret this?
<img width="824" alt="Screenshot 2021-03-09 at 13 59 52" src="https://user-images.githubusercontent.com/15104192/110482648-cb7e1700-80e0-11eb-82f8-34a6b6564ae8.png">

2) Looking at **nucleosome_signal**, **pct_reads_in_peaks** and **blacklist_ratio**, I'm noting one cluster has a very different / low value compared to the others. Am I correct in assuming this cluster is likely low QC cells and would be best removed?

<img width="821" alt="Screenshot 2021-03-09 at 13 59 33" src="https://user-images.githubusercontent.com/15104192/110483276-7989c100-80e1-11eb-90a5-342311990a44.png">
<img width="824" alt="Screenshot 2021-03-09 at 13 59 59" src="https://user-images.githubusercontent.com/15104192/110482883-0c762b80-80e1-11eb-816e-455c1cfff3b3.png">
<img width="825" alt="Screenshot 2021-03-09 at 14 00 05" src="https://user-images.githubusercontent.com/15104192/110482895-1009b280-80e1-11eb-8ec1-593ea82dd42c.png">

Any help here would be appreciated

**Comment by timoast** [comment, marked as answer]
It looks like you have a group of low-quality cells in the center of the plot that have high blacklist-region counts, low percent reads in peaks, low nucleosome banding pattern, and low total counts. I would recommend adjusting some of the QC thresholds so that these cells are filtered out, then re-processing the dataset without those cells.
